src/java.net.http/share/classes/jdk/internal/net/http/HttpClientImpl.java

```diff
@@ -126,6 +127,16 @@ final class HttpClientImpl extends HttpClient implements Trackable {
     static final long IDLE_CONNECTION_TIMEOUT_H2 = getTimeoutProp("jdk.httpclient.keepalive.timeout.h2", KEEP_ALIVE_TIMEOUT);
     static final long IDLE_CONNECTION_TIMEOUT_H3 = getTimeoutProp("jdk.httpclient.keepalive.timeout.h3", IDLE_CONNECTION_TIMEOUT_H2);
 
+    static final UseVTForSelector USE_VT_FOR_SELECTOR =
+        Utils.useVTForSelector("jdk.internal.httpclient.tcp.selector.useVirtualThreads", "default");
```

Comment:
Hello Daniel, I vaguely remember we discussed whether or not to use a value called "default" for this property. But I don't remember what we decided. Looking at this now, I am wondering whether we should just do something like:
```java
System.getProperty("jdk.internal.httpclient.tcp.selector.useVirtualThreads", "true")
```
to keep it simple?